src/actions/counter.js

```diff
@@ -11,3 +13,11 @@ export const counterIncrement = () => ({
 export const counterDecrement = () => ({
   type: Actions.Decrement
 });
+
+export const counterIncrement2 = () => ({
```

Comment:
This definitely works and will be fine for this phase but not for phase 3. There is a proverb in software engineering that you either do something 0, 1 or N times. If you're doing something greater then 1 time then you should think about how to make it work for any N. Once again, fine for this phase but think about how are you gonna create and manage a dynamic list of counters for phase 3.